dev-python/ciso8601/ciso8601-2.1.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Fast ISO8601 date time parser for Python written in C"
+HOMEPAGE="https://github.com/closeio/ciso8601 https://pypi.org/project/ciso8601/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
```

Comment:
If you add IUSE="test" then you must have to add RESTRICT="!test? ( test )" because we don't want to run test every time but just when it is needed

Reply:
This comment is for every ebuilds you have made